### ISSUE TITLE
Stopped working, can't connect anymore

### ISSUE THREAD

**Issue by Aspage** [travisghansen/hass-pfsense#159]
It used to work flawlessly. Suddenly, it stopped working.

When I try to setup the integration to my PFSense now, I get this in the logbook:

Unexpected err=<ProtocolError for [redacted]:[redacted]@192.168.121.254/xmlrpc.php: 405 Not Allowed>, type(err)=<class 'xmlrpc.client.ProtocolError'>

I've tried searching for it on here and using Google, of course, but can't seem to find the problem.

Using 2.6.0. Tried admin user and a new user. Same outcome. Can't even connect.

What can I try or do?

**Comment by travisghansen** [comment]
Sounds like maybe the ip has changed or something else? Seems really odd to just stop working.

**Comment by Aspage** [comment]
No, IP hasn't changed. I actually even reinstalled PFSense completely (had to for other reasons) and on new installation doesn't connect either.

**Comment by thomaspriv** [comment]
Same issue here :/

**Comment by M0WGF** [comment]
Have the same issue.   It will not login. 

I get SSL error 

if I uncheck the check SSL 

I get unexpected error instead. 

IP, login details haven't changed.  All worked until up Hassio update 2023.5.1 I think.

**Comment by M0WGF** [comment]
I get the following errors in the debug logs. 

First error with SSL Check enabled the second error with it disabled. 

Unexpected err=AbortFlow('Flow aborted: already_configured'), type(err)=<class 'homeassistant.data_entry_flow.AbortFlow'>
22:00:30 – (ERROR) pfSense (custom integration)
Unexpected get_system_info error err=SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1007)'), type(err)=<class 'ssl.SSLCertVerificationError'>
22:00:25 – (ERROR) pfSense (custom integration)

**Comment by M0WGF** [comment]
Oh thank you so much I had completely missed the disabled integration.  All fixed 👍

**Comment by REHCTELF-FLETCHER** [comment]
Hey I’m having the same issue, but with a quick google search I discovered that there’s something called a xmlrpc.php attack aka a ddos attack and I wondering if the constant query’s sent to pfsense is causing nginx to get overloaded.
